cluster-data/alonzo-genesis-increased-exUnits.yaml

```diff
@@ -0,0 +1,363 @@
+# Originally taken from https://hydra.iohk.io/build/7578887/download/1/index.html
+
+lovelacePerUTxOWord: 34482
+executionPrices:
+  prSteps:
+    numerator: 721
+    denominator: 10000000
+  prMem:
+    numerator: 577
+    denominator: 10000
+maxTxExUnits:
```

Comment:
We only care about `maxTxExUnits`, right? Why not make it into a parameter?

Reply:
We care about these 4:
```
maxTxExUnits:
  exUnitsMem: 10000000000000
  exUnitsSteps: 10000000000000000
maxBlockExUnits:
  exUnitsMem: 50000000000000
  exUnitsSteps: 40000000000000000
```
The problem is we just feed the file to the PParams and don't deal with any of it's information so we'd have to rewrite the file every time we spawned the cluster to the parameters we set. IMO this introduces more complexity than its worth.

Reply:
solved as per @jy14898 suggestion in [cdf6623](https://github.com/mlabs-haskell/plutip/pull/165/commits/cdf6623a11947a991d7e322ac6eb7e9959874987)